nifi-extension-bundles/nifi-box-bundle/nifi-box-processors/src/main/java/org/apache/nifi/processors/box/GetBoxFileCollaborators.java

```diff
@@ -60,10 +61,10 @@
 @SeeAlso({FetchBoxFile.class, ListBoxFile.class})
 @WritesAttributes({
         @WritesAttribute(attribute = ID, description = ID_DESC),
-        @WritesAttribute(attribute = "box.collaborations.<status>.users.ids", description = "Comma-separated list of user collaborator IDs by status"),
-        @WritesAttribute(attribute = "box.collaborations.<status>.groups.ids", description = "Comma-separated list of group collaborator IDs by status"),
-        @WritesAttribute(attribute = "box.collaborations.<status>.users.emails", description = "Comma-separated list of user collaborator emails by status"),
```

Comment:
So... technically speaking, removing those could be considered as a breaking change. Unless the RC1 of 2.3.0 is cancelled, this processor is going out with 2.3.0. Can we keep those anyway?